### ISSUE TITLE
So sorry to reach out to you through here!

### ISSUE THREAD

**Issue by derto42** [eduardonn/camera-system-object-detection#1]
Unfortunately github doesn't have messages so I hope this is okay.

I am looking for anyone who has shown recent interest in PyQt5.

Me and a couple of other people are working on a productivity tool using PyQt5. Our goal is to create something that hundreds of people would find useful. The whole thing uses PyQt5 so it has been challenging to find devs that know how to use it. Check it out, if you are interested, we need all the help we can get. This could be a good opportunity to learn something new :)
 
https://github.com/derto42/FlowBuddy

Here is my discord as well if interested: https://discord.gg/ahvpBbXG

**Comment by eduardonn** [comment]
Hey! No problem at all.

This project was actually made as my college's final project. It was my first contact with most of the technologies in it and I can't say I had the best experience with PyQt5, haha.

But I'll definitely check that project out and maybe in the future I'll be able to help!

**Comment by derto42** [comment]
Of course. If the projects sounds interesting come join our discord, maybe you could give it a try at the very least when we make a full release.
